containers/home/Home.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from "react";
+import {
+  StatusBar,
+  TouchableOpacity,
+  Text,
+  View,
+  FlatList
+} from "react-native";
+import { Colors } from "../../colors/Colors";
+import { createStackNavigator, createAppContainer } from "react-navigation";
+import { Item } from "../../components/item/Item";
+import { ButtonCustom } from "../../components/buttonCustom/ButtonCustom";
+import { styles } from "./styles";
+
+class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: [
+        {
+          "first-description": "Description",
```

Comment:
What do you think about reworking a bit the model for each todo? Perhaps making it something like:
```js
{
  id: [auto-generated-id],
  title: [title],
  description: [description],
  completed: [true|false],
}
```

Reply:
Instead of name the inner state `data`, what do you think about naming it `todos`?

Reply:
I agree with these changes.